.github/workflows/backend.yml

```diff
@@ -0,0 +1,103 @@
+name: Backend
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+      - 'fix**'
+      - 'ci**'
+    tags:
+      - '**'
+    paths:
+    - 'src/metabase/**'
+    - 'modules/drivers/**'
+    - 'enterprise/backend/**'
+    - '**/project.clj'
+    - '**/deps.edn'
+    - '.github/workflows/**'
+
+jobs:
+
+  be-linter-bikeshed:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Get M2 cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/project.clj') }}
```

Comment:
Maybe you need different cache keys for different linters:

https://github.com/metabase/metabase/pull/14701/checks?check_run_id=1849943293

```
Post job cleanup.
Unable to reserve cache with key Linux-m2-8d6f5f693bbf38ab33791318f1e8d6af8df4abee705ad213372ce8e848ac4d34, another job may be creating this cache.
```

either way the different linters will have slightly different dependencies